### ISSUE TITLE
Spice Test: Reorganize under subfolders to mimic /lib

### ISSUE THREAD

**Issue by moollaza** [duckduckgo/zeroclickinfo-spice#2433]
Some of our Perl modules live in subdirectories. However, almost none of tests for those modules live in corresponding subdirectories.

We recently updated DuckPAN and the `duckpan new` command to now create test files under the same sub-directories as their related Perl modules:

For example `duckpan new Foo::Bar` will create `/lib/DDG/Spice/Foo/Bar.pm` and `t/Foo/Bar.t`
 
This way the tests are logically organized instead of all dumped into a single directory. I think we should cleanup our existing `t/ ` so that it more closely mimics the organization of `/lib`.

Any thoughts?

/cc @duckduckgo/community-leaders

**Comment by GuiltyDolphin** [comment]
@moollaza I agree that there should be consistency and perhaps we should move the tests to these folders.

I've been working on this locally but I think I need to send a fix to duckpan before we perform this (tests aren't running for directories).
